### DISCUSSION TITLE
Infosec Warning: Hidden executable in NTFS alternate data stream

### DISCUSSION THREAD

**Discussion by mnaric01** [mockoon/mockoon#1621] [Help] [answered]
Has anyone come across this warning from their infosec teams? We are wondering if this is just the nature of the installer or if our scan tool is confused due to foreign characters in the installer's path.

**Comment by 255kb** [comment, marked as answer]
This was never reporterd and I would say it's a false positive. 
When doing a `dir /R` using cmd.exe, the only Alternate Data Stream I see is `Zone.Identifier` that Windows automatically add to all downloads:
![image](https://github.com/user-attachments/assets/4efedba9-38ca-4955-85dd-466c81555219)

Unless you see something more concerning I guess you can ignore it.

**Reply by mnaric01** [reply]
Thank you!
